<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>downloadPopUp</class>
 <widget class="QWidget" name="downloadPopUp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(196, 236, 236);</string>
  </property>
  <widget class="QFrame" name="downloadPopUp_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>250</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>500</width>
     <height>600</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background: qlineargradient(
    x1: 0, y1: 0, x2: 1, y2: 1,
    stop: 0 rgba(50, 50, 50, 1), stop: 1 rgba(128, 128, 128, 1)
);
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="logoButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <underline>false</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: none;
border: none;
color: black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../images/icon.png</normaloff>../images/icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>90</width>
      <height>90</height>
     </size>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>360</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
background: white;
border-radius: 10%;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="importButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>360</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Arial&quot;;
color: blue;
</string>
     </property>
     <property name="text">
      <string>Drag the file or chose from device</string>
     </property>
    </widget>
    <widget class="QLabel" name="wrongLabel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>360</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 9pt &quot;Arial&quot;;
color: red;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="finishButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>210</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Arial&quot;;
color: white; 
background: qradialgradient(
    cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
    radius: 1.35, stop: 0 rgba(25, 81, 68, 1), stop: 1 rgba(57, 148, 127, 1)
);

border-radius: 10%;</string>
    </property>
    <property name="text">
     <string>Finish</string>
    </property>
   </widget>
   <widget class="QLabel" name="selectFileLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>400</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 20pt &quot;Arial&quot;;
color: white;
background: none;</string>
    </property>
    <property name="text">
     <string>Select your .txt file</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
